templates/rpm.erb

```diff
@@ -60,13 +53,18 @@ AutoProv: yes
 <% end -%>
 # Seems specifying BuildRoot is required on older rpmbuild (like on CentOS 5)
 # fpm passes '--define buildroot ...' on the commandline, so just reuse that.
-BuildRoot: %buildroot
-<% if !prefix.nil? and !prefix.empty? %>
+#BuildRoot: buildroot
```

Comment:
Any reason why you are not using the macro expansion for buildroot as it previously was?

Reply:
Nevermind, i just noticed you actually commented it. Makes sense to remove the macro % prefix then